### DISCUSSION TITLE
How to do link prediction (multiple graphs)

### DISCUSSION THREAD

**Discussion by hiraphor** [danielegrattarola/spektral#345] [Q&A]
Hi - is there an example of how to do link prediction with Spektral (in particular when training on multiple graphs)? Very lost on how to approach this. Thanks :)

Edit: In particular, I have a dataset with many small graphs with node features, edge features, and adjacency matrices. I'm trying to essentially predict these adjacency matrices using Spektral (ie link prediction). Could anyone advise on how to do this?

**Comment by danielegrattarola** [comment]
Hi,

You can find an example here: https://github.com/FilippoMB/Variational-Graph-Auto-encoders-Tensorflow-2-Spektral-

Cheers
